library/bignum_mod_raw.h

```diff
@@ -155,7 +155,24 @@ int mbedtls_mpi_mod_raw_write( const mbedtls_mpi_uint *A,
 /* END MERGE SLOT 4 */
 
 /* BEGIN MERGE SLOT 5 */
-
+/**
+ * \brief Perform a known-size modular addition.
+ *
+ * Calculate `A + B modulo N` where \p A, \p B, and \p N have the same size.
+ *
+ * \param[out] X    The result of the modular addition.
+ * \param[in] A     Little-endian presentation of the left operand. This
+ *                  must be smaller than \p N, and have the same number of
+ *                  limbs.
+ * \param[in] B     Little-endian presentation of the right operand. This
+ *                  must be smaller than \p N, and have the same number of
+ *                  limbs.
+ * \param[in] N     The address of the modulus.
+ */
+void mbedtls_mpi_mod_raw_add( mbedtls_mpi_uint *X,
+                              mbedtls_mpi_uint const *A,
+                              mbedtls_mpi_uint const *B,
```

Comment:
(should fix) in Mbed TLS we generally prefer
```suggestion
                              const mbedtls_mpi_uint *B,
```

as on the next line (there are a few of these, just marking this one)